src/App.tsx

```diff
@@ -1,40 +1,38 @@
-import { useEffect, useState } from "react";
+import { useState } from "react";
 import TransferButtons from "./TransferButtons";
 import List from "./List";
 import {
-  calculateWordCount,
-  debounce,
   intersection,
   not,
 } from "./utils";
+import { ListInput } from "./components/list-input/ListInput";
+import './App.css';
 
-function App() {
-  const [inputValue, setInputValue] = useState("");
-  const [currWordCount, setCurrWordCount] = useState(0);
-  const [firstListItems, setFirstListItems] = useState([
-    "one",
-    "two",
-    "three",
-    "four",
-  ]);
-  const [secondListItems, setSecondListItems] = useState([]);
+export type ListItem =  { 
+  id: string; 
+  name: string; 
+};
+
```

Comment:
Better to have id for each item, because items can have same names if user decides to.  Then, you will have bugs in your code.

Reply:
Makes sense 👍